.devcontainer/clamav-almalinux/Dockerfile

```diff
@@ -0,0 +1,116 @@
+RUN dnf -y install epel-release && dnf config-manager --set-enabled crb
+
+# Install necessary packages
+RUN dnf -y --allowerasing install \
+    cmake \
+    bison \
+    check \
+    curl \
+    flex \
+    gcc \
+    gcc-c++ \
+    git \
+    gdb \
+    glibc-all-langpacks \
+    make \
+    man-db \
+    net-tools \
+    psmisc \
+    pkg-config \
+    python3-pip \
+    python3-pytest \
+    sudo \
+    tcpdump \
+    valgrind \
+    wget \
+    zip \
+    bzip2-devel \
+    check-devel \
+    curl-devel \
+    json-c-devel \
+    sendmail-devel \
+    ncurses-devel \
+    pcre2-devel \
+    openssl-devel \
+    libxml2-devel \
+    zlib-devel \
+    && rm -rf /var/cache/apt/archives
+
+RUN echo "REMOTE_USER is ${REMOTE_USER} and REMOTE_UID is ${REMOTE_UID}"
+
+# Create a non-root user
+RUN echo "Creating user ${REMOTE_USER} with UID ${REMOTE_UID}" && \
+    adduser --uid "$REMOTE_UID" "$REMOTE_USER" && \
+    echo "${REMOTE_USER} ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/${REMOTE_USER} && chmod 0440 /etc/sudoers.d/${REMOTE_USER}
+
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y \
+    && . $CARGO_HOME/env \
+    && rustup update
+
+# Build ClamAV
+RUN mkdir -p ./build && cd ./build \
+    && cmake .. \
+    -DCARGO_HOME=$CARGO_HOME \
+    -DCMAKE_BUILD_TYPE="Release" \
+    -DCMAKE_INSTALL_PREFIX="/usr" \
+    -DCMAKE_INSTALL_LIBDIR="/usr/lib" \
+    -DAPP_CONFIG_DIRECTORY="/etc/clamav" \
+    -DDATABASE_DIRECTORY="/var/lib/clamav" \
+    -DENABLE_CLAMONACC=OFF \
+    -DENABLE_EXAMPLES=OFF \
+    -DENABLE_JSON_SHARED=ON \
+    -DENABLE_MAN_PAGES=OFF \
+    -DENABLE_MILTER=ON \
+    -DENABLE_STATIC_LIB=OFF \
+    && make DESTDIR="/clamav" -j$(($(nproc) - 1)) install \
+    && rm -r /clamav/usr/include /clamav/usr/lib/pkgconfig/ \
+    && sed -e "s|^\(Example\)|# \1|" \
+    -e "s|.*\(LocalSocket\) .*|\1 /tmp/clamd.sock|" \
+    -e "s|.*\(TCPSocket\) .*|\1 3310|" \
+    -e "s|.*\(TCPAddr\) .*|#\1 0.0.0.0|" \
+    -e "s|.*\(User\) .*|\1 clamav|" \
+    -e "s|^\#\(LogFile\) .*|\1 /var/log/clamav/clamd.log|" \
+    -e "s|^\#\(LogTime\).*|\1 yes|" \
+    /clamav/etc/clamav/clamd.conf.sample > /clamav/etc/clamav/clamd.conf \
+    && sed -e "s|^\(Example\)|# \1|" \
+    -e "s|.*\(DatabaseOwner\) .*|\1 clamav|" \
+    -e "s|^\#\(UpdateLogFile\) .*|\1 /var/log/clamav/freshclam.log|" \
+    -e "s|^\#\(NotifyClamd\).*|\1 /etc/clamav/clamd.conf|" \
+    -e "s|^\#\(ScriptedUpdates\).*|\1 yes|" \
+    /clamav/etc/clamav/freshclam.conf.sample > /clamav/etc/clamav/freshclam.conf \
+    && sed -e "s|^\(Example\)|# \1|" \
+    -e "s|.*\(MilterSocket\) .*|\1 inet:7357|" \
+    -e "s|.*\(User\) .*|\1 clamav|" \
+    -e "s|^\#\(LogFile\) .*|\1 /var/log/clamav/milter.log|" \
+    -e "s|^\#\(LogTime\).*|\1 yes|" \
+    -e "s|.*\(ClamdSocket\) .*|\1 unix:/tmp/clamd.sock|" \
+    /clamav/etc/clamav/clamav-milter.conf.sample > /clamav/etc/clamav/clamav-milter.conf
+# && ctest -V --timeout 3000
```

Comment:
For me, ctest also fails on Debian, though only the `clamd` ones and with this error:
```
4: ERROR: Wed Aug  6 18:14:15 2025 -> TCP: Cannot bind to [localhost]:3319: Address already in use
4: ERROR: Wed Aug  6 18:14:15 2025 -> Cannot set socket permission for clamd-test.socket to 666
```

Unsure what that's all about. Weirdly, AlmaLinux devcontainer ones fail all over the place. Especially the valgrind ones.  Very strange. 

I'm not really sure what I'm saying here.  I'm just as happy not having it build clamav at all as I am having it run the tests.  So I guess I'm happy with this even though it is different from the Debian one.

Reply:
To be honest, I don't think we should be building clamav in the dockerfile for the dev container. I think the the point of a dev container is to provide a development environment so you can do the build as desired. I'm inclined to remove the clamav build steps entirely from both Dockerfiles.